### PR TITLE
Remove empty chunks on TRUNCATE hypertable

### PR DESCRIPTION
**Pull request by olofr**
Depends on PR #127

### CODE REVIEW

File: sql/timescaledb--0.1.0--0.2.0.sql [hunk 1]
```diff
@@ -521,6 +521,31 @@ CREATE OR REPLACE FUNCTION _timescaledb_internal.timescale_trigger_names()
 $BODY$
    SELECT array['_timescaledb_main_insert_trigger', '_timescaledb_main_after_insert_trigger'];
 $BODY$;
+
+
+CREATE OR REPLACE FUNCTION _timescaledb_internal.truncate_hypertable(
```
Comment (cevian) [inline sql/timescaledb--0.1.0--0.2.0.sql]: This change is wrong. should not modify old update file

File: test/expected/truncate_hypertable.out [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+ 1257894002000000000 | dev1      |      2.5 |        3 |          | 
+ 1257897600000000000 | dev1      |      4.5 |        5 |          | f
+ 1257987600000000000 | dev1      |      1.5 |        1 |          | 
+ 1257987600000000000 | dev1      |      1.5 |        2 |          | 
+ 1257894000000000000 | dev2      |      1.5 |        1 |          | 
+ 1257894000000000000 | dev2      |      1.5 |        2 |          | 
+(12 rows)
+
+TRUNCATE "two_Partitions";
+NOTICE:  index "1-two_Partitions_device_id_timeCustom_idx" does not exist, skipping
```
Comment (cevian) [inline test/expected/truncate_hypertable.out]: Can we supress these NOTICES?

File: test/expected/truncate_hypertable.out [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+ Schema | Name | Type | Owner 
+--------+------+------+-------
+(0 rows)
+
+\set ON_ERROR_STOP 1
+SELECT * FROM "two_Partitions";
+ timeCustom | device_id | series_0 | series_1 | series_2 | series_bool 
+------------+-----------+----------+----------+----------+-------------
+(0 rows)
+
```
Comment (cevian) [inline test/expected/truncate_hypertable.out]: Can we also add a `\d+ "two_Partitions"` to check that it has no inherited children

**Comment by olofr** [comment]
Your comments should be fixed now @cevian

**Comment by RobAtticus** [comment]
Can you change the commit message to match the title I made? I think this is more in line with our commit msg style (i.e. using the imperative form)?

**Comment by olofr** [comment]
Commit message updated @RobAtticus.

**Pull request merged**